packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_AlgebraicOverlappingOperator_def.hpp

```diff
@@ -94,12 +94,13 @@ namespace FROSch {
                                                                             MapPtr repeatedMap)
     {
         if (this->Verbose_) std::cout << "WARNING: Can we just copy the pointers like that without changing the matrix...?\n";
+
         this->OverlappingMap_ = repeatedMap;
         this->OverlappingMatrix_ = this->K_;
-        
         for (int i=0; i<overlap; i++) {
             ExtendOverlapByOneLayer(this->OverlappingMatrix_,this->OverlappingMap_);
```

Comment:
Yes, that's correct. We should migrate to Ifpack2 for this.

Reply:
In fact, I would not want FROSch to use a function from Ifpack2 because we don't want a dependency on Ifpack2. Also, we probably have to introduce slightly more sophisticated ways to determine the overlap for certain saddle point problems. Therefore, we will probably not be able to use exactly the same code that Ifpack2 uses. 
However, I believe that the Ifpack2 implementation is more efficient at the moment since it only uses the graph of the matrix. We definitely should do this as well.